### ISSUE TITLE
Backend Error

### ISSUE THREAD

**Issue by dwi4122** [TharunKumarrA/FC-TeamForge#22]
Deploy the mysql database to the vercel project. No players are showing in the deployed project.

<img width="1440" alt="Screenshot 2024-12-28 at 2 35 31 pm" src="https://github.com/user-attachments/assets/8e52cefd-31e7-4170-8902-a37bdac412d8" />

**Comment by TharunKumarrA** [comment]
ok @dwi4122, the backend repo for now is missing the schema.sql file that will create all the tables for storing player details. The file will be pushed by eod and then you can run the backend locally. With my vercel plan I cant have the backend running in deployment. Hope you understand

**Comment by TharunKumarrA** [comment]
There was an issue created to address the same issue in the backend repo, in the mean time if you are comfortable replicating the database locally in the machine I am attaching the snaps of the table structure

**Comment by TharunKumarrA** [comment]
and @dwi4122 if you are willing you can take up the issue in the backend repo for database initialization and grab that bounty ;)

**Comment by thecodingvivek** [comment]
@dwi4122 I have got the same issue. After going through the code finally i understood that :

1)No need to deploy the sql database on vercel because as we are working on this project locally just try to install mysql . you can checkout this link to install :  https://www.youtube.com/watch?v=BxdSUGBs0gM  

2)create tables with schemas
***example in mysql command line***
![image](https://github.com/user-attachments/assets/d3239354-3687-4e95-96fe-0d984ff289e3)

3)add database credentials in ***sqlconnect.js***
![image](https://github.com/user-attachments/assets/e1b96932-ad37-4860-9c3c-43d78da18826)

4)Create an account in ngrok. https://ngrok.com/  and copy the ***authtoken***  and  add it your .env file
***in server.js***
![image](https://github.com/user-attachments/assets/7554bbb3-6e3a-47dc-9e5a-bddd20f0a1e9)

5)Try running the server You should should see Something like this in your terminal
![image](https://github.com/user-attachments/assets/d395dbe3-011b-4e5f-a1cb-208d2e00d328)

6)Now add the url shown in the terminal to your ***constants.js*** file
![image](https://github.com/user-attachments/assets/6045e573-72b2-43b3-9f60-c9c80d5f1388)

7)You will now get a cors error so add header to allow requests from all origins: res.header("Access-Control-Allow-Origin", "*");
![image](https://github.com/user-attachments/assets/0622522f-bb3c-4980-83ee-1e249b43c707)


8)Your fine to go now.***HOPE THIS HELPS :)***

**Comment by TharunKumarrA** [comment]
Yeah for now this is how to access backend. Great explanation @thecodingvivek

**Comment by thecodingvivek** [comment]
@TharunKumarrA Anna can i share this comment on backed repo for database initialisation to solve this issue ..?

**Comment by TharunKumarrA** [comment]
Yeah sure that would help
